src/main/resources/portal.properties.EXAMPLE

```diff
@@ -106,8 +106,11 @@ ldap.attributes.last_name=sn
 ldap.attributes.given_name=givenName
 ldap.attributes.email=mail
 
+# study view settings
 # always show studies with this group
 always_show_study_group=
+mdacc.heatmap.study.meta.url=http://bioinformatics.mdanderson.org/study2url?studyid=
```

Comment:
@ChrisWakefield yes, CDSA has the same issue. They are working on setting up a https server.

It's probably OK for now since our public portal is on http, but in the future, we may want to upgrade all our instances to https.